os/board/rtl8730e/src/component/os_dep/osdep_service_misc.h

```diff
@@ -17,19 +17,21 @@
 #ifndef __OSDEP_SERVICE_MISC_H_
 #define __OSDEP_SERVICE_MISC_H_
 
-// #define printk				printf
-// #define DBG_ERR(fmt, args...)		printf("\n\r[%s] " fmt, __FUNCTION__, ## args)
-
-// #define HALT()				do { cli(); for(;;);} while(0)
-#undef ASSERT
-#define ASSERT(x)			do { \
+#define RTK_HALT()   \
+	do {         \
+		cli();   \
+		for (;;) \
+			;    \
+	} while (0)
+#undef RTK_ASSERT
+#define RTK_ASSERT(x)			do { \
```

Comment:
is it not nessesary to change to tizen lite assertion?

Reply:
Hi @ewoodev , BSP normally just call `assert_param`, which has been linked to Tizen Lite assert operation. Thus, I renamed this to prevent from naming conflict. Currently, no code invoke RTK_ASSERT.